apps/condo/domains/resident/tasks/helpers/sendResidentsNoAccountNotifications.js

```diff
@@ -35,14 +35,16 @@ const prepareAndSendNotification = async (context, resident, period) => {
 
     // TODO(DOMA-3376): Detect locale by resident locale instead of organization country.
     const country = get(resident, 'residentOrganization.country', conf.DEFAULT_LOCALE)
+    const tin = get(resident, 'residentOrganization.tin')
     const locale = get(COUNTRIES, country).locale
     const notificationKey = makeMessageKey(period, resident.property.id, resident.id)
     const organizationId = get(resident, 'residentOrganization.id')
+    const url = `${conf.SERVER_URL}/payments/addaccount/?residentId=${resident.id}&categoryId=${CATEGORY_ID}&organizationTIN=${tin}`
```

Comment:
Is it ok if tin may be `'undefined'`?
<img width="249" alt="image" src="https://user-images.githubusercontent.com/25384290/228445873-180084e4-ad4d-41c0-86e0-e42c68fa8341.png">

Reply:
Organizations always have tin

![image](https://user-images.githubusercontent.com/91055869/228470827-5f9d133c-4845-4348-aee4-ad18b3f91992.png)